src/api/getBaldiyaByCodeForWilaya.js

```diff
@@ -0,0 +1,38 @@
+const projectBaladiya = require('../utils/projections/projectObject');
+
+const getBaldiyaByCodeForWilaya = (data) =>
+/**
+   * Takes a wilaya code (mattricule) and a baladiya code  and return a baladiya
+   *
+   * @example Get the baladiya of adrar  in adrara(mattricule:1 ,code: 101)
+   *
+   *
+   * getBaldiyaByCodeForWilaya(1, 101)
+   *
+   * @param { Number } mattricule wilaya code (mattricule)
+   * @param { Number } code baladiya code (code)
```

Comment:
Please check `jsdoc` here and in all changed files

```suggestion
   * @param { number } mattricule wilaya code (mattricule)
   * @param { number } code baladiya code (code)
```